srcpkgs/rset/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'rset'
+pkgname=rset
+version=2.1
+revision=1
+build_style=configure
+make_install_args="PREFIX=/usr"
+make_check_target=test
+hostmakedepends="gcc"
+checkdepends="ruby>=2.4 wget"
+short_desc="Configure systems using using any scripting language"
+maintainer="Benjamin Lee <benjamin@typ3.tech>"
+license="GPL-3.0-or-later"
```

Comment:
rset itself (build and installed binaries) uses the ISC license

Reply:
Ah, good catch. Fixed